Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationLoad.h

```diff
@@ -41,7 +41,7 @@ class MANTID_ALGORITHMS_DLL CorelliPowderCalibrationLoad
 
   /// Extra help info
   const std::vector<std::string> seeAlso() const override {
-    return {"CorelliPowderCalibrationGenerate "
+    return {"CorelliPowderCalibrationCreate "
```

Comment:
Should be
```C
return {"CorelliPowderCalibrationGenerate", "CorelliPowderCalibrationDatabase", "CorelliPowderCalibrationApply"};
```
so the names can be resolved into links in the user documentation.

Reply:
I'm reading now `std::vector<std::string>` and realized I parked my brain in a far far away place :facepalm: